stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/StreamAttachmentFactories.kt

```diff
@@ -34,18 +35,32 @@ public object StreamAttachmentFactories {
      */
     private const val DEFAULT_LINK_DESCRIPTION_MAX_LINES = 5
 
+    /**
+     * Instructs the Giphy to fill all of the available space.
+     */
+    public const val GIPHY_FILL_MAX_SPACE: Float = -1f
```

Comment:
Pixel for pixel gifs can be quite small. This way we allow users to let the gifs fill max space if they want to, our preserve better visual quality by making them pixel faithful.

Btw. if you have a better place for this like somewhere in `ChatTheme`, please let me know.